### DISCUSSION TITLE
Several questions about verifying the transfer free energy of styrene

### DISCUSSION THREAD

**Discussion by rswf** [ricalessandri/Martini3-small-molecules#52] [Q&A] [answered]
Hello developers! I am modeling polystyrene using Martini3 force fields and want to screen for the right type of beads based on the octanol-water partition coefficient of styrene or oligomers of styrene.
1) Can you tell me how to determine the number of solvent molecules in a water-saturated octanol and pure water system? I noticed the effect of the number of solvents on the free energy of solvation in the TI method.
2) I would like to know what is the octanol-water free energy of transfer for styrene in a database of small molecules? I found different experimental values for partition coefficients in my search, what is the value to use as a reference?
3) May I know if the all-atom model for octanol is also obtained with LigParGen and uses the 1.14 CM1A-LBCC charge?

**Comment by ricalessandri** [comment]
Hi @rswf! First of all, please note that there's a PS model shipped with [polyply](https://github.com/marrink-lab/polyply_1.0), namely: https://github.com/marrink-lab/polyply_1.0/blob/master/polyply/data/martini3/PS.martini3.ff.

To answer your questions:
1. From the [small mol paper](https://doi.org/10.1002/adts.202100391): "_Note that we simulate hydrated octanol, that is we add a ≈0.3 mole fraction of water according to experimental conditions._[Ref.](https://doi.org/10.1039/FT9959100649)"
2. I've been mostly using values from [this paper](https://doi.org/10.1021/ci400112k); for styrene the logP value is 2.95.
3. For the record, I have not computed all-atom logPs in the [small mol paper](https://doi.org/10.1002/adts.202100391). However, if you want to do that, indeed you should be consistent: that is, if you generate a OPLS/LigParGen/1.14 CM1A-LBCC force field for styrene, then you want to do the same for the octanol. *Unless*, there's an optimized and well-validated model of octanol (like for water, e.g., the TIP3P water model that is compatible with OPLS). However, I am not aware of any such model for octanol.

**Reply by rswf** [reply]
@ricalessandri Thanks for your answer! I did miss the polystyrene Martini3 model that was already available.

Regarding question 1, I did notice this mentioned in the paper, what I wanted to know more was the size of the system for calculating the transfer free energy. For example, if a single styrene molecule is modeled in 2000 water molecules, should the number of solvent molecules in the corresponding hydrated octanol system also be 2000? Or should I keep the volume of hydrated octanol the same as 2000 water molecules? Please let me know the size of the solvent system used in your construction of the small molecule database.

Also, when I run a simulation using the styrene Martini3 model from the database, the stepwise coupled styrene-water molecule interactions give free energy values greater than zero (the opls force field gives negative results), which seems to imply that the styrene molecule prefers to be in the vacuum rather than in water. May I ask if there is a problem with my results?	

out hydrated octanol (1480 beads, 480 octanol, 40 water) : 8.105  +-  0.139   kJ/mol	      
                                                      into water (1480 beads) : 13.130  +-  0.154  kJ/mol	 
                                                                                       ΔG : 21.235  kJ/mol

**Reply by ricalessandri** [reply, marked as answer]
@rswf sorry for the slow follow up. These are my DG_solvation and DG_transfer data for the Martini 3 STYR model. Note that W-->0_CG means the DG of solvation from W to vacuum, etc.:

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/alessandri/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/alessandri/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">

</head>

<body link="#467886" vlink="#96607D">


LABEL | W-->0_CG | HD-->0_CG | OCO-->0_CG | CLF-->0_CG | HD-->W_CG | OCO-->W_CG | CLF-->W_CG
-- | -- | -- | -- | -- | -- | -- | --
STYR | -9.238 | 4.861 | 6.805 | 12.547 | 14.099 | 16.043 | 21.785



</body>

</html>

In terms of the absolute value of DG of solvation, do not compare CG and AA. Martini is not meant to reproduce DG of solvation. Martini cannot reproduce DG of solvation due to the use of the Lennard-Jones potential, which limits the parameter space. Instead, compare only the DG of transfer between different solvents - Martini is parametrized to match those.

Looking at your data, something is off, in particular with the water solvation free energy (~4 kJ/mol difference w.r.t. mine). 

The size of the solvent boxes does not matter, as long as they are *large enough* that the molecule is at infinite dilution (i.e., cannot interact with itself). You can check that by making sure that it doesn't "see" its periodic image (look at the Martini cutoffs). For the record, I'd use solvent boxes with starting dimensions of  5.25 x 5.25 x 5.25 nm^3. Setting them up with `gmx solvate`.

**Reply by rswf** [reply]
@ricalessandri Thank you very much for your reply and the reference data provided！

1, When I calculated the free energy of solvation for styrene and 1480 W beads, the equilibrated box size is 5.52263nm 5.52263nm 5.52263nm and the cutoff is 1.1nm.  I think the box is large enough.

2, When I use the gmx solvate command  `gmx solvate -cp CG-PS1.gro -cs solvent.gro -box 5.25 5.25 5.25` to build the box of styrene and W beads, the box is filled with 125 W beads, and the calculated free energy of solvation in this system is -9.148 +- 0.372 kJ/mol (water to vacuum) (the box dimensions are 2.41652nm 2.41652nm 2.41652nm) and this looks like it to be correct. But when the number of W beads increases to 1480, the free energy of solvation becomes -12.888 +- 0.166 kJ/mol (water to vacuum).

This is what confuses me, in my test results the value of the solvation free energy is correlated with the number of solvents or the size of the simulation box.

**Reply by ricalessandri** [reply]
@rswf 1) Yes, the 5.5^3 nm^3 box is large enough. 2) This should not happen; there must be something else going on. Maybe the 2.4^3 nm^3 box is too small, and you're getting the correct value (the value that matches mine) in that case by chance? This free energy should just depend on the interactions between PS1 and what it can interact with (which means, what surrounds the PS1 molecule up to the cutoff). All else does not matter. To debug this, I'd try to play with different box sizes (3.5^3 nm^3, 4.5^3 nm^3, etc.), different thermodynamic integration point simulation lengths, and different solvents (do you observe this only with water? What about chloroform or hexadecane, etc.).

**Reply by rswf** [reply]
@ricalessandri As you say, the solvation free energy should not correlate with either the number of solvent molecules or the size of the simulation box in the case of a sufficiently large box size.

For the system of styrene and chloroform, free energy values of about 14 kJ/mol were obtained in both the 5.66 nm and 3.94 nm cubic boxes.

For the system of styrene and hydrated octanol, free energy values of about 8 kJ/mol were obtained in both the 4.88 nm and 3.89 nm cubic boxes.

For the system of styrene and water, free energy values of about 12.9 kJ/mol were obtained in both the 5.53 nm and 4.69 nm cubic boxes.

The problem should be in my calculations. I downloaded the force field file from the Martini force field official website, and the topol file for styrene from the Martini3-small-molecules project, so maybe there is something wrong with my simulation parameter settings? I will attach my mdp file, please bother to correct my mistake, also can you provide your mdp file for my reference?

[free-energy.zip](https://github.com/user-attachments/files/15750694/free-energy.zip)

**Reply by rswf** [reply]
After comparing my mdp file with the one from the tutorial on the Martini website, I found that I had DispCorr enabled in the CG simulation, causing an error in the numerical calculation of the transfer free energy. There is no problem with the small molecule modeling of styrene. I apologize for the trouble caused by my oversight.

Thanks to @ricalessandri  for your detailed answers and the data!

**Reply by ricalessandri** [reply]
@rswf Great that you found the problem! Thanks for the follow-up.
